src/components/Sheet/AudacityManager.tsx

```diff
@@ -103,7 +108,8 @@ function AudacityManager(props: IProps) {
     audacityManagerSelector,
     shallowEqual
   );
-  const fetchUrl = useFetchUrlNow();
+  const ts: ISharedStrings = useSelector(sharedSelector, shallowEqual);
+  const fetchUrl = useFetchUrlNow(ts.expiredToken);
```

Comment:
Do we need to pass in ts.expiredToken? can we just return this value and check if it is the return value?